packages/playground/website/src/components/error-report-modal/index.tsx

```diff
@@ -50,6 +51,15 @@ export function ErrorReportModal() {
 		resetSubmission();
 	}
 
+	function getContext() {
```

Comment:
Also screen resolution?

Reply:
I can add `window.innerWidth/Height`. This could help with debugging UI issues.

Reply:
Done https://github.com/WordPress/wordpress-playground/pull/1253/files#diff-46008bd779d2dd5cb639272c824aa9529721d171731bc22c6b073c0d73aab3c0R60